.github/workflows/check-sidecar-tasks.sh

```diff
@@ -2,7 +2,7 @@
 set -eu
 
 make sidecar-tasks
-if ! git diff --quiet deploy/ods-pipeline/charts/tasks/templates; then
```

Comment:
Keep in mind that this whole script can go away, right? No need for sidecar tasks anymore ... ;)

Reply:
To have a better overview of what still needs to be done we try going through the changes step by step (according to the list we came up with with you). That is why there might be changes currently in this PR that we need to remove again as soon as we tacke another bullet point (e.g. *"Remove sidecar variants (incl make target) and render in yaml like with deployment resources"*)

Reply:
OK good, just wanted to make sure that is still on your radar. I also think the list we came up with is basically unordered, so you might want to do some steps (like removing sidecars) before the others to avoid work? Up to you!